maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -433,6 +433,8 @@ void cli( CliRequest cliRequest )
 
         if ( cliRequest.commandLine.hasOption( CLIManager.VERSION ) )
         {
+            // MNG-7032: Also disable colours if in batch mode
+            disableColorsInLogfileOrBatch( cliRequest );
```

Comment:
Please move this line to line 427. I can imagine we're going to add colors to the help output too. And maybe there will be more options in the future. This disabling a global thing, should not be restricted to `version`.